pipelines/rj_cvl__osinfo_pdf_reconstruct/tasks.py

```
@@ -0,0 +1,291 @@
+    Args:
+        items: List to split
+        chunk_size: Size of each chunk
+
+    Returns:
+        List of chunked lists
+
+    Raises:
+        ValueError: If chunk_size is less than or equal to 0
+    """
+    if chunk_size <= 0:
+        raise ValueError(f"chunk_size must be greater than 0, got {chunk_size}")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)